.travis.yml

```diff
@@ -12,13 +12,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Travis continuous integration system configuration file. 
+# Travis continuous integration system configuration file.
 #
 # Overview on how to customize the configuration is located at:
 # http://docs.travis-ci.com/user/customizing-the-build/
 
 # Explicitly set the environment to be container-based. This makes the builds faster.
```

Comment:
Update this comment and say why sudo is needed (so that future modifiers know why this was introduced).